modules/troubleshooting-disabling-autoreboot-mco-console.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco-console_{context}"]
+= Disabling Machine Config Operator from automatically rebooting by using the console
+
+To avoid unwanted disruptions from changes made by the Machine Config Operator (MCO), you can use the {product-title} web console to modify the machine config pool (MCP) to prevent the MCO from making any changes to nodes in that pool. This prevents any reboots that would normally be part of the MCO update process. 
+
+[NOTE]
+====
+Pausing a machine config pool stops all updates to your {op-system} nodes, including updates to the operating system, security, certificate, and any other updates related to the machine config. Pausing should be done for short periods of time only.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+To pause or unpause automatic MCO update rebooting: 
+
+* Pause the autoreboot process:
+
+. Log into the {product-title} web console as a user with the `cluster-admin` role.
+
+. Click *Compute* -> *Machine Config Pools*.
+
+. On the *Machine Config Pools* page, click either *master* or *worker* depending upon which nodes you want to pause rebooting.
+
+. On the *master* or *worker* page, click *YAML*.
+
+. In the YAML field, update the `spec.paused` field to `true`.
++
+.Sample `MachineConfigPool` object 
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+...
```

Comment:
The preview build renders an odd underline beneath the ellipsis. Maybe that's intentional or known and I'm just not aware.